elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/engine/SQLEntityHydrator.java

```diff
@@ -68,7 +71,8 @@ public SQLEntityHydrator(
                 .getResultList();
 
         return loaded.stream()
-                .map(obj -> new AbstractMap.SimpleImmutableEntry<>((Object) getEntityDictionary().getId(obj), obj))
+                .map(obj -> new AbstractMap.SimpleImmutableEntry<>(
+                        getEntityDictionary().getValue(obj, "id", null), obj))
```

Comment:
getValue is not a safe way to get the ID of an entity.  Why did we change this?

Reply:
Get id is always returning String.

Reply:
The keyword "id" is not safe in this context.  This won't work for entities that have remapped the ID field to something named other than ID.  We need to use getId.